js/main.js

```diff
@@ -4,6 +4,145 @@ const hamburger = document.querySelector('.hamburger');
 const closeIcon = document.querySelector('.closeIcon');
 const menuIcon = document.querySelector('.menuIcon');
 
+const appProjects = [
+  {
+    title: 'First Project',
+    picture: 'images/works/tonic/Cover1.svg',
+    content: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley',
+    tech1: 'html',
+    tech2: 'css',
+    tech3: 'javascript',
+    live: '<a href="https://www.google.com">See Live <img src="images/live.svg" alt=""></a>',
+    source: '<a href="https://www.google.com">See Source <img src="images/Vector.svg" alt=""></a>',
```

Comment:
- [ ] I would also suggest that you remove the anchor tags around the live link and the source link to help keep your code DRY so you don't have multiple anchor tag calls and keep them like this:
```
link: "link_to_live_version.com"
source:  "link_to_source_version.com"
```
- [ ] This way, you can only have one anchor tag in your template string below and reference the links similar to what you did for the pictures in your project :wink: 
- [ ] Kindly repeat this throughout your code :100: